src/modules/position_estimator_inav/position_estimator_inav_main.c

```diff
@@ -311,19 +311,30 @@ int position_estimator_inav_thread_main(int argc, char *argv[])
 		{ 0.0f },		// D (pos)
 	};
 
-	float corr_sonar = 0.0f;
-	float corr_sonar_filtered = 0.0f;
+	float corr_lidar = 0.0f;
+	float corr_lidar_filtered = 0.0f;
 
 	float corr_flow[] = { 0.0f, 0.0f };	// N E
 	float w_flow = 0.0f;
 
-	float sonar_prev = 0.0f;
+	float lidar_prev = 0.0f;
 	//hrt_abstime flow_prev = 0;			// time of last flow measurement
-	hrt_abstime sonar_time = 0;			// time of last sonar measurement (not filtered)
-	hrt_abstime sonar_valid_time = 0;	// time of last sonar measurement used for correction (filtered)
+	hrt_abstime lidar_time = 0;			// time of last lidar measurement (not filtered)
+	hrt_abstime lidar_valid_time = 0;	// time of last lidar measurement used for correction (filtered)
+
+	//----------test--------------------------
+	float flow_test[] = { 0.0f, 0.0f };
```

Comment:
@ChristophTobler Could you remove your test variables, I guess they are not used anymore, right?